pkg/logs/status/status_test.go

```diff
@@ -60,12 +60,43 @@ func TestStatusDeduplicateWarnings(t *testing.T) {
 	assert.ElementsMatch(t, []string{"Unique Warning"}, status.Warnings)
 }
 
+func TestStatusDeduplicateErrors(t *testing.T) {
+	defer Clear()
+	createSources()
+
+	AddGlobalError("bar", "Unique Error")
+	AddGlobalError("foo", "Identical Error")
+	AddGlobalError("foo", "Identical Error")
```

Comment:
What should happen if the message changes and the key remains the same ? this could happen when we inject arguments into the log message

Reply:
The messages are unique per key so only the most recent warning/error from the key will be displayed. 

For errors, as errors are stopping the agent, I think that's fine as your latest error (and only error) will display the correct info about why the agent stopped (with or without arguments in the log message).

Reply:
Sounds good